.github/workflows/apply.yaml

```diff
@@ -0,0 +1,30 @@
+name: Apply GitHub labels
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
@lfdebrux I'm not sure what this line does and I'm finding the [Github docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) a bit difficult to understand. Can you explain why this is needed?

Reply:
It's not strictly necessary, it just makes it easier to test the action. The `workflow_dispatch` event basically means that you can start the workflow manually from the GitHub GUI or CLI. Unfortunately it won't appear until this PR is merged though.

Reply:
It's explained better in https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow